README.md

```diff
@@ -65,6 +65,47 @@ On **Windows** use `;` instead of `:` in -cp:
 java -cp lib/*;out/production/jmavsim.jar me.drton.jmavsim.Simulator
 ```
 
+#### Key command ####
+
+Views:
+    F    - First-person-view camera.
```

Comment:
This does not properly render on github: have a look at the preview here: https://github.com/stmoon/jMAVSim/blob/402532912ff2562085852e2a3735b1a726d43057/README.md.
You can use a list for these, like this:
```
- F    - First-person-view camera.
```

Reply:
Thanks @bkueng 

I modify the README according to markdown syntax